queries.sql

```diff
@@ -53,4 +53,29 @@ GROUP BY species;
 SELECT species, AVG(escape_attempts) 
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-GROUP BY species;
\ No newline at end of file
+GROUP BY species;
+
+-- Alter to add new column
+ALTER TABLE animals
+ADD COLUMN species varchar(255);
+
+-- transaction 1
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species = 'unspecified';
+COMIT;
+
+-- transaction 2
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
```

Comment:
- [ ] in this transaction after deleting the table you're supposed to verify if the table got deleted.
- [ ] then after rollback verify if it was restored.